lib/rust/profiling/src/lib.rs

```diff
@@ -71,6 +72,8 @@ pub struct Metadata {
     pub profiling_level: ProfilingLevel,
     /// Label of the measurement..
     pub label:           String,
+    /// Aggregate statistics for various frame metrics, collected over the time of the measurement.
+    pub rendering:       Option<frame_stats::Summary>,
 }
```

Comment:
the statistics are not only about rendering. Like WASM memory usage is about the whole WASM program, so this should not be named `rendering`.

Reply:
This is a name taken verbatim from the design doc's JSFiddle example, and the example did contain a `wasmMemoryUsage` sub-field inside the `rendering` field. Given this, seeing that you want this changed, do you want me to change the name in the resulting JSON as well? or do you want me to use a different field name here, but keep the design doc naming in JSON via `#[serde(rename = "rendering")]`?